core/modules/script.py

```diff
@@ -1,15 +1,52 @@
+from viur.core import Module, skeleton
 from viur.core.bones import *
 from viur.core.prototypes.tree import Tree, TreeSkel
-from viur.core import db, utils, conf, skeleton, tasks
+from viur.core import db, utils, conf, skeleton, tasks, i18n
 from viur.core.prototypes.tree import Tree
 import re
+import os
 
 
 # pre-compile patterns for vfuncs
 DIRECTORY_PATTERN = re.compile(r'^[a-zA-Z0-9äöüÄÖÜ_-]*$')
 FILE_PATTERN = re.compile(r'^[a-zA-Z0-9äöüÄÖÜ_-]+?.py$')
 
 
+def _get_modules():
+    res = {}
+
+    for key in dir(conf["viur.mainApp"].vi):
+        module = getattr(conf["viur.mainApp"].vi, key, None)
+        if not isinstance(module, Module):
+            continue
+
+        if admin_info := module.describe():
+            res[key] = admin_info
+
+    return res
+
```

Comment:
This function must be moved to another place.
What about `viur.module`? In perspective, the function should be solved other than doing a dir() on the vi-renderer.